src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -3,12 +3,61 @@
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
+import exceptions.RegistrationException;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_AGE = 18;
+    private static final int MAX_AGE = 150;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        checkForNull(user);
+        return (isValidValue(user)
+                && storageDao.get(user.getLogin()) == null) ? storageDao.add(user) : null;
+    }
+
+    public void checkForNull(User user) {
+        if (user != null) {
+            if (user.getLogin() == null) {
+                throw new RegistrationException("Login can't be null!");
+            }
+            if (user.getPassword() == null) {
+                throw new RegistrationException("Password can't be null!");
+            }
+            if (user.getAge() == null) {
+                throw new RegistrationException("Age can't be null!");
+            }
+            return;
+        }
+        throw new RegistrationException("Object of user can't be null!");
+    }
+
+    public boolean isValidValue(User user) {
```

Comment:
Please use consistent approach.
Either all validations are void or boolean.
This will make understanding of code easier